projects/plugins/jetpack/_inc/client/components/button/index.jsx

```diff
@@ -48,6 +53,20 @@ export default class Button extends React.Component {
 			this.domNode = node;
 		};
 
-		return React.createElement( element, props, this.props.children );
+		// Open external links in new window.
+		if ( isLink && isExternalLink ) {
+			props.target = '_blank';
+		}
+
+		return React.createElement(
+			element,
+			props,
+			<>
+				{ this.props.children }
+				{ isLink && isExternalLink && (
+					<Gridicon className="dops-card__link-indicator" icon="external" />
```

Comment:
Would it make sense to bring the styles for `dops-card__link-indicator` into the Button component, so everything is in one place?